gcn/notices/icecube/single_neutrino_alerts.example.json

```diff
@@ -13,6 +20,6 @@
   "healpix_url": "https://roc.icecube.wisc.edu/public/",
```

Comment:
Do you intend to have this link to an individual healpix map associated with this event, or just this general link?  The actual map would be more useful.

Reply:
Updating to include a full example that's available online